src/WebUI/Components/KubeSummary.tsx

```diff
@@ -191,29 +192,24 @@ export class KubeSummary extends BaseComponent<IKubeSummaryProps, IKubernetesCon
                     deploymentList={this.state.deploymentList || {} as V1DeploymentList}
                     replicaSetList={this.state.replicaSetList || {} as V1ReplicaSetList}
                     key={format("dc-{0}", this.state.namespace || "")}
-                    onItemInvoked={this._onDeploymentItemInvoked}
+                    onItemActivated={this._onDeploymentItemInvoked}
                 />
                 <DaemonSetListingComponent
                     daemonSetList={this.state.daemonSetList || {} as V1DaemonSetList}
-                    key={format("ds-list-{0}", this.state.namespace||"")}
+                    key={format("ds-list-{0}", this.state.namespace || "")}
                 />
-                <StatefulSetListingComponent 
+                <StatefulSetListingComponent
                     statefulSetList={this.state.statefulSetList || {} as V1StatefulSetList}
-                    key={format("sts-list-{0}", this.state.namespace||"")}
+                    key={format("sts-list-{0}", this.state.namespace || "")}
                 />
 
                 {this.state.podList && this.state.podList.items && this.state.podList.items.length > 0 && this.getOrphanPods()}
             </PivotItem>
         );
     }
 
-    private _onDeploymentItemInvoked = (item?: any, index?: number, ev?: Event) => {
-        this.setState({
-            showDeployment: true,
-            selectedItem: item,
-            showService: false,
-            showSummary: false
-        });
+    private _onDeploymentItemInvoked = (event: React.SyntheticEvent<HTMLElement>, tableRow: ITableRow<any>) => {
+        // TODO: Handle this after we introduce flux structure, as this requires selected DeploymentItem information which exists only with DeploymentsComponent as of now

```

Comment:
please check my comment in deploymentscomponent, ideally we should have workloads compnent and servicescomponent, rather deploymentscomponent and servicescomponent